bitwarden_license/src/Scim/Utilities/ScimConcurrencyRateLimiter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.RateLimiting;
+using Bit.Core.AdminConsole.Authorization;
+
+namespace Bit.Scim.Utilities;
+
+public static class ScimConcurrencyRateLimiter
+{
+    public const string PolicyName = "ScimConcurrencyLimiter";
+
+    public static IServiceCollection AddScimConcurrencyRateLimiter(this IServiceCollection services) =>
+        services.AddRateLimiter(options => options.AddPolicy(PolicyName, context =>
+            RateLimitPartition.GetConcurrencyLimiter(context.GetOrganizationId(),
```

Comment:
❓ `CurrentContext` isn't available at this point? Lot easier than the extension method.